Test.C

```diff
@@ -117,70 +115,126 @@ int main(){
             {3, attackStock[0]},
     };
 
+    //declare
+    struct Player player;
+    int currentRound = 0;
+
     /* TODO: RANDOMLY POPULATE ROUNDS WITH ATTACKS (Random number generator to pull from stock?) */
 
-    //initialize player
-    Player player;
-    printf("Enter Player Name: ");
-    scanf("%s", &player.name);
+    //Setup player
     player.equipmentInventory[0] = 1; /* TODO: RANDOMLY POPULATE EQUIPMENT (Do we need to tier equipment?) */
     player.encryptionInventory = {encryptionStock[0]}; /* TODO: RANDOMLY POPULATE ENCRYPTION (Do we need to tier encryption?) */
     player.currentCredits = 30;
 
-    while(player has options and more rounds exists){
-        //receive credits NATHAN
+    //output pregame information
+    displayBriefingMessage(&player);
+
+    while(currentRound < NUM_ROUNDS){
+        printf("|================================== Round %2d ==================================|\n", currentRound);
+
+        //output player status (if statements)
+        //current credit
         printf("Credits: %d\n", player.currentCredits); //current credit
-		
+        
         int i;
         for (i = 0; i < NUM_EQUIPMENT; i++){
             // details about the equipment type itself is stored in equipmentStock
             // amount of this equipment type that the player has is stored in equipmentInventory
             printf("%s: %d\n", equipmentStock[i]->name, player.equipmentInventory[i]);
         } //current equipment
-            
-		//pulls from info from index amount table referencing index struct table
-            
 
-struct EncryptionInventory encryptionPointer = *player.encryptionInventory;
-		while (encryptionPointer != null){
+	      //pulls from info from index amount table referencing index struct table
+
+
+        struct EncryptionInventory encryptionPointer = *player.encryptionInventory;
+	      while (encryptionPointer != null){
             // details about the encryption type itself is stored in encryptionStock
             // amount of this encryption type that the player has is stored in encryptionInventory
             printf("%s\n", encryptionPointer->encryption->name);
             encryptionPointer=encryptionPointer->next;
         }  //current encrypt
         //iterate and print through current encryption
-    }
-        
-        //receive credits
+
+        //receive credits NATHAN
         //use equation to figure out how many credit user receives
         //tell user how credits received and what encrypt method was used
```

Comment:
```suggestion
int sum;
int newCredits;
printf("Original credits: %d\n", player.currentCredits); //prints out the current credits the player has
		
//use equation to figure out how many credit user receives
newCredits = player.currentCredits * Encryption.effectiveness * Equipment.messageAmount;
		
for (int i = 0; i < equipment; i++){
	sum += equipmentStock[i] * equipmentInventory[i]; //sum message amount of equipment(s)
}
		
//multiply the sum player.encryptionInventory.encryption.effectiveness; 
	//TODO: figure out which encryption was chosen
newCredits = sum * player.encryptionInventory.encryption.effectiveness; //number of credits player receives
		
player.currentCredits += newCredits; //adding credits to currentCredits
		
//tell user how credits received and what encrypt method was used
printf("You received %d credits!\n", newCredits); //telling the user how many credits they were awarded
		
printf("Credits: %d\n", player.currentCredits); //current credit
```

Reply:
Is this already in the "Added Equation for Calculating Credits" pull request?